### DISCUSSION TITLE
Request to join your team

### DISCUSSION THREAD

**Discussion by krishnaspeace** [ROM-builders/temporary#22611] [General]
Good day sir, I am here again to join team, and also you answer me last time and need to #learn, can you check my device tree once if it's qualified now, i just force push my tree and make it clean, once check please,since it is asking for need approval from many days it's hard for me to build roms and i am student so i don't have that mush time that i wait for approval i think you understand my condition .

Successful Builds:
Pixel Experience (Unofficial): https://cirrus-ci.com/build/6520555906531328
Pixel Experience Plus (Unofficial): https://cirrus-ci.com/build/4624937730703360
Evolution X (Unofficial): https://cirrus-ci.com/build/5225825516126208

Device tree: https://github.com/krishnaspeace/android_device_xiaomi_ysl
Telegram I'd: https://t.me/krishnaspeace

I hope i will be qualified this time, thank you for providing ROM-builders

**Comment by Apon77** [comment]
I really want to add people to the team. There is no discrimination at all. But you must be eligible by completing the #learn instruction. Otherwise, I really can't help. There is nothing personal here. And there is no need for a paid server to follow #learn instructions. Please follow that. I would like to help you always only if you are eligible. 
Thank you.

**Reply by krishnaspeace** [reply]
Sir please clear what I need to learn which thing left
